elastic/integrations
# [kibana] Simplify ECS formatted logs ingest pipelines

## What does this PR do?

Closes https://github.com/elastic/integrations/pull/4047

This PR simplifies the logs ingest pipelines, since they are in ECS format, reusing what's has been done for the [Platform Observability package ](https://github.com/elastic/integrations/tree/main/packages/platform_observability)


## How to test this PR locally

- Build the elasticsearch package: `cd packages/kibana && elastic-package build`
- Start a stack: `elastic-package stack up -v -d --version 8.5.0-SNAPSHOT`
- start the elasticsearch service: `cd packages/kibana && elastic-package service up -v`
- Install the Kibana package
  - Set audit log path: /tmp/service_logs/audit.log
  - Set audit log path: /tmp/service_logs/kibana.log
- Check the Logs UI
